docs/architecture/grpc-for-wcf-developers/error-handling.md

```diff
@@ -62,9 +62,9 @@ catch (RpcException)
 > [!IMPORTANT]
 > When you provide additional metadata for errors, be sure to document the relevant keys and values in your API documentation, or in comments in your `.proto` file.
 
-## gRPC Richer Error Model
+## gRPC richer error model
 
-Looking ahead, Google has developed a [richer error model](https://cloud.google.com/apis/design/errors#error_model) that is more like WCF's [FaultContract](xref:System.ServiceModel.FaultContractAttribute), but isn't supported in C# yet. Currently, it's only available for Go, Java, Python and C++, but support for C# is expected to come next year.
+Google has developed a [richer error model](https://cloud.google.com/apis/design/errors#error_model) that's more like WCF's [FaultContract](xref:System.ServiceModel.FaultContractAttribute), but this model isn't supported in C# yet. Currently, it's only available for Go, Java, Python, and C++.
```

Comment:
Why remove the mention that C# support is upcoming?

Reply:
It's a legal issue. Technical documentation in our organization covers what is, not what might be at a future time. The reason is that customers can assume a promise is being made.